File: Java/noDuplicates.java
import java.util.ArrayList;
import java.util.Scanner;

public class noDuplicates {
    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);
        String input = sc.nextLine();

        ArrayList<String> words = new ArrayList<>();
        String[] splitted = input.split(" ");
        boolean found = false;

        for(int i = 0; i < splitted.length; i++){
            if(!words.contains(splitted[i])){
                words.add(splitted[i]);
            }
            else{
                System.out.println("no");
                found = true;
                break;
            }
        }

        if(!found){
            System.out.println("yes");
        }
    }
}

File: C#/No_Duplicates.cs
using System.IO;
using System;
using System.Collections;

class Program
{
    static void Main()
    {   
       string line = Console.ReadLine();
       string[] subs = line.Split(' ');
       ArrayList myAL = new ArrayList();
       
       int i = subs.Length-1;
       bool found_duplicate = false;
       
       // Working the other way, back to front, but works just as well:
       while(i >= 0){
           string current = subs[i];
           if(myAL.Contains(current)){
               found_duplicate = true;
               Console.WriteLine("no");
               break;
           }
           
           myAL.Add(current);
           i--;
       }
        
       if(!found_duplicate){
           Console.WriteLine("yes");
       }
    }
}
